database/migrations/2018_10_07_120133_fix_json_column.php

```diff
@@ -20,21 +20,15 @@ public function up()
 
         $databasename = $connection->getDatabaseName();
 
-        $columns = $connection->table('information_schema.columns')
-                            ->select('table_name', 'column_name')
-                            ->where([
-                                ['table_schema', '=', $databasename],
-                                ['data_type', '=', 'json'],
-                            ])
-                            ->whereIn('table_name', [
-                                'default_life_event_types',
-                                'life_event_types',
-                                'life_events',
-                            ])
-                            ->get();
+        $columns = DB::select(
```

Comment:
This too: why we need to rewrite this part?

Reply:
See above comment, the same thing applies here.